src/bp/ui-studio/src/web/components/Shared/ElementsList/InputElement.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import classnames from 'classnames'
+import style from './style.scss'
+
+export class InputElement extends React.Component {
+  elementInputRef = React.createRef()
+
+  state = {
+    error: undefined
+  }
+
+  onEnter = value => {
+    this.props.onEnter(value)
+  }
+
+  handleOnEnter = event => {
+    const inputValue = event.target.value
+    const enterKeyPressed = event.keyCode === 13
+    const inputNotEmpty = inputValue !== ''
+    const elementUnique = !this.props.elements.includes(inputValue)
+
+    if (enterKeyPressed) {
```

Comment:
can be replaced with if(event.key === 'Enter') instead of checking for the numeric code